### DISCUSSION TITLE
Too few arguments to function Filament\Support\Actions\BaseAction::make()

### DISCUSSION THREAD

**Discussion by lovemintblue** [filamentphp/filament#3066] [Help]
### Package

filament/filament

### Package Version

v2.13.24

### Laravel Version

9.2

### Livewire Version

v2.0.0

### PHP Version

ph8.0

### Bug description

list page ,Too few arguments to function Filament\Support\Actions\BaseAction::make(), filament/filament/src/Resources/Pages/ListRecords.php on line 102 and exactly 1 expected

**Comment by danharrin** [comment]
Send your code

**Reply by lovemintblue** [reply]
> 发送您的代码

This problem occurred when I upgraded from V2.13.23 to V2.13.24，  V2.13.23 is no problem

**Reply by danharrin** [reply]
Please share your code and Flare exception URL

**Reply by lovemintblue** [reply]
no code,  I create new project，execute  php artisan make:filament-resource Customer command, list is error, you can try。edit、create is ok;

**Reply by lovemintblue** [reply]
> Please share your code and Flare exception URL

 php artisan make:filament-resource User  url: http://127.0.0.1:8000/admin/users       error:

![image](https://user-images.githubusercontent.com/72596738/178105459-e115aaa8-d9b1-4721-88d4-96e55451d5a0.png)

**Reply by lovemintblue** [reply]
> Please share your code and Flare exception URL

I didn't write any code， Only appear v2.13.24 version,

**Reply by lovemintblue** [reply]
"filament/filament": "^2.13.24",

**Reply by lovemintblue** [reply]
I'm not sure if it's a bug, trouble you！

**Reply by danharrin** [reply]
I think there is some opcache issue on your end, `BaseAction::make()` does not have any required arguments

**Reply by lovemintblue** [reply]
> I think there is some opcache issue on your end, `BaseAction::make()` does not have any required arguments

I don't what to do.I very strange ,I had to use v2.13.23 version.   wait and see if anyone else has the same problem as me, finally, thank you!!

**Reply by danharrin** [reply]
Can you visit the BaseAction class and paste the `make()` method here

**Reply by lovemintblue** [reply]
> Can you visit the BaseAction class and paste the `make()` method here
![WechatIMG104](https://user-images.githubusercontent.com/72596738/178233630-567b749b-0bda-4985-916a-eb186773acca.jpeg)


    /**
     * @deprecated Actions are no longer pre-defined.
     */
    protected function getEditAction(): Tables\Actions\Action
    {
        return Tables\Actions\EditAction::make('edit');
    }

make() changed to make('edit') is ok.   why?  php8.0

**Reply by lovemintblue** [reply]
make(). Cannot get default values，

**Reply by danharrin** [reply]
No... look at the `public static function make()` in `BaseAction.php`, not when we use it.

**Reply by lovemintblue** [reply]
![WechatIMG107](https://user-images.githubusercontent.com/72596738/178237040-f6ea8939-3ffc-4b0b-8732-fde96dcc3af3.jpeg)
![WechatIMG108](https://user-images.githubusercontent.com/72596738/178237089-82431fc7-89ce-4d59-a3c7-82cbb5b73097.jpeg)
 $name no default value ,  $name = ' ' is ok,  please confirm it。

**Reply by lovemintblue** [reply]
![WechatIMG109](https://user-images.githubusercontent.com/72596738/178238588-9c47b9f2-04bf-4565-9ade-2e21d782675e.jpeg)
102 make() ，unable to execute

**Reply by lovemintblue** [reply]
you can update your demo project,  update to v2.13.24 version ， see if it's ok;

**Reply by danharrin** [reply]
Your code is out of date 🤷 

https://github.com/filamentphp/filament/blob/2.x/packages/support/src/Actions/BaseAction.php#L30-L44

**Reply by danharrin** [reply]
Can you send your composer.json please

**Reply by lovemintblue** [reply]
thank you!  Local Composer mirroring problem，
